api_parser.py

```diff
@@ -0,0 +1,34 @@
+import requests
+from datetime import datetime, timedelta
+
+
+def MakeApiRequest(start_date):
+    '''
+    This function works as follows:
+    1. Generate daily list of dates from start_date
+    2. For every day it makes api request to get USD to EUR exchange rate
+    3. After receiving a response it saves it to file
+
+    :param start_date: start date
+    :type start_date: int
+
+    :return None:
+    '''
+    # Generate daily date from start date
+    print('Start parsing ...')
+    start = datetime.strptime(start_date, '%Y-%m-%d')
+    end = datetime.now()
+    days = (end - start).days
+    for diff in range(days):
```

Comment:
for date in range(days)(

Reply:
It is really a diff in days between starting and processing dates, so `date` is the best name for this variable, but anyway - this is MR with worst possible python code I can imagine for educational purposes - note repository name)